docs/copied-from-beats/shared-systemd.asciidoc

```diff
@@ -5,6 +5,11 @@ The DEB and RPM packages include a service unit for Linux systems with
 systemd. On these systems, you can manage {beatname_uc} by using the usual
 systemd commands.
 
+ifdef::apm-server[]
+We recommend that the {beatname_lc} process is run as a non-root user.
+Therefor, that is the default setup for {beatname_uc}'s DEB package and RPM installation.
+endif::apm-server[]
+
 ==== Start and stop {beatname_uc}
 
 Use `systemctl` to start or stop {beatname_uc}:
```

Comment:
@graphaelli, any reason why we omit `sudo` from the start/stop/enable/disable commands?

Reply:
good question, I'm good with adding `sudo` to make things more copy and paste friendly.